homeassistant/helpers/script.py

```diff
@@ -615,7 +615,10 @@ def __init__(
         self,
         hass: HomeAssistant,
         sequence: Sequence[Dict[str, Any]],
-        name: Optional[str] = None,
+        name: str,
+        domain: str,
+        *,
+        description: Optional[str] = "script",
```

Comment:
I think that the name `description` is ambigious. It could be interpreted as the description of the script/automation action.

Reply:
Maybe "object_description", and maybe add a comment above it on how it will be used?

Reply:
How about `running_description` with an added comment?

Reply:
```python
        # Used in "Running <running_description>" log message
        running_description: Optional[str] = None,
```